msteams-platform/graph-api/rsc/test-resource-specific-consent.md

```diff
@@ -74,8 +109,10 @@ Execute the entire permissions collection for each API call. The permissions tha
 
 ## Test revoked RSC permissions using [Postman](https://www.postman.com/)
 
-1. Uninstall the app from the specific team.
-2. Follow the steps for [Test added RSC permissions using Postman](#test-added-rsc-permissions-using-the-postman-app).
+1. Uninstall the app from the specific resource.
+2. Follow the steps for either chat or team: 
+    1. [Test added RSC permissions to a team using Postman](#test-added-rsc-permissions-to-a-team-using-the-postman-app).
+    2. [Test added RSC permissions to a chat using Postman](#test-added-rsc-permissions-to-a-chat-using-the-postman-app).
 3. Check all the response status codes to confirm that the specific API calls, **succeeded, have failed with an HTTP 403 status code**.
```

Comment:
Something is wrong here.

Check all the response status codes to confirm that the specific API calls **have failed with an HTTP 403 status code** because the permissions got successfully revoked.